### ISSUE TITLE
Question re Interactive Highlight for Solute Adjusters

### ISSUE THREAD

**Issue by terracoda** [phetsims/membrane-transport#385]
It's currently, possible for me to get an Interactive Highlight around the entire Solute Adjuster with no highlight on any of the buttons when I hover my mouse in between the Arrow buttons.

I think we have 3 choices:
1. Never have a the outer highlight for mouse overs.
2. Ensure one only sees the outer highlight when the hover on a button is actually true. 
3. Show a thinner outer pink highlight and inner think one for mouse users, and still only show both when option 2 is true.

Which option makes the most sense here. I think having the highlights appear in sequence will cause issues - like failed activations on appearance of the outer highlight

@brettfiedler , what do yo think? 

Tagging @samreid and @jessegreenberg for your thoughts, too. We might have the mismatch for many composite controls when it comes to Interactive Highlights.

**Comment by jessegreenberg** [comment]
I went ahead and committed that because there is an easy option for it. You can try it out on main if you like. The bad behavior can be observed in a recent dev version. But if we need another option, let me know.

**Comment by terracoda** [comment]
Working nicely for pointer now without the outer highlight. Closing.
